src/pages/waypoint/tutorials/[collectionSlug].tsx

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Related to my previous comment in `src/pages/waypoint/tutorials/[...tutorialSlug]/index.tsx`, you'll need to export a `getStaticProps` function here that returns a `product` object in `props` in order to get the `ProductSwitcher` working on this page.

Reply:
Sounds good, I think i'll leave these two pages without the product switcher for now until I set up data fetching for them.

Reply:
That works! 💖